packages/components/src/components/circular-progress/circular-progress.tsx

```diff
@@ -25,7 +25,8 @@ const CircularProgress = ({
 }: TCircularProgressProps) => {
     const normalizedRadius = radius - stroke / 2;
     const circumference = normalizedRadius * 2 * Math.PI;
-    const strokeDashoffset = circumference - (progress / 100) * circumference;
+    // eslint-disable-next-line no-bitwise
+    const strokeDashoffset = circumference - (~~progress / 100) * circumference;
```

Comment:
Are we trying to solve a NaN problem with the bitwise operator? :) could you please elaborate a bit on this🙏🏻